libraries/opensk/src/ctap/ctap1.rs

```diff
@@ -156,6 +160,22 @@ impl TryFrom<&[u8]> for U2fCommand {
     }
 }
 
+fn to_uncompressed(public_key: &impl EcdsaPublicKey) -> [u8; 1 + 2 * EC_FIELD_BYTE_SIZE] {
```

Comment:
Here is a case where you want to refer to the trait by name. There's an alternative option if you prefer:

```rust
use crate::api::crypto::ecdsa::{self, PublicKey as _, ...};

fn to_uncompressed(public_key: &impl ecdsa::PublicKey) -> ...
```

Reply:
I like the explicit usage. We have too many ecdsa etc. in use, so the longer type is useful to understand the code. In that case, the anonymous trait import is nice.

Reply:
We ideally shouldn't have too many `ecdsa` because one (`opensk::api::crypto::ecdsa`) is shadowing the other (`crypto::ecsda`). Both are only visible when wiring them together.

Reply:
I think that's the case. We just have two different `PublicKey` (as expected).